server/scripts/k6service.js

```diff
@@ -0,0 +1,46 @@
+import http from 'k6/http';
+import { sleep, check } from 'k6';
+import { Rate } from 'k6/metrics';
```

Comment:
Seems like you decided not to use Rate in the script. Do you still need this import statement?

Reply:
Good call. I might use it again later but did not use it this time around so I'm going to take it out.